commit d94be7f55608188023c095e4aa7bc98d9682b5e4
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-19T14:27:59-07:00 (3 months ago)
    
    TRITON-1827 mark these as 'release' components in jr metadata: triton-grafana

diff --git a/tools/jr-manifest.json b/tools/jr-manifest.json
index 1496b92..67fdd26 100644
--- a/tools/jr-manifest.json
+++ b/tools/jr-manifest.json
@@ -920,6 +920,7 @@
         {
             "name": "triton-grafana",
             "labels": {
+                "release": true,
                 "tritonservice": "grafana",
                 "mg": "grafana",
                 "vm": true
